src/main/java/mate.academy/Main.java

```diff
@@ -10,6 +11,7 @@ public class Main {
     public static void main(String[] args) {
         // Please test your Injector here. Feel free to push this class as a part of your solution
         Injector injector = Injector.getInjector();
+        @Inject
         ProductService productService = null;
         List<Product> products = productService.getAllFromFile("products.txt");
```

Comment:
Attempting to call a method on a `null` reference will result in a `NullPointerException`. Ensure that `productService` is properly instantiated by the injector before calling methods on it.